src/executables/util_cs.c

```diff
@@ -3258,20 +3255,21 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
   do
     {
+      memset (local_database_name, 0x00, CUB_MAXHOSTNAMELEN);
+      strcpy (local_database_name, database_name);
+      strcat (local_database_name, "@localhost");
 
       if (check_applied_info)
 	{
-	  memset (local_database_name, 0x00, CUB_MAXHOSTNAMELEN);
-	  strcpy (local_database_name, database_name);
-	  strcat (local_database_name, "@localhost");
-
 	  db_clear_host_connected ();
 
+	  printf ("\n *** Applied Info. *** \n");
 	  if (check_database_name (local_database_name))
 	    {
 	      goto check_applied_info_end;
 	    }
-	  if (db_login ("DBA", NULL) != NO_ERROR)
+	  error = db_login ("DBA", NULL);
```

Comment:
```suggestion

	  error = db_login ("DBA", NULL);
```

Reply:
I will apply it than you. Are `db_logn()` and `db_restart()` in the same category(same prefix)? so there is not space(\n) between them?

Reply:
I just recommend that it would be nice if there is a new line after if statement.

Reply:
Okay I got it. I will apply to behind the db_restart() 'if' statement.